indico/migrations/versions/20220203_1524_f046e456629f_make_building_optional.py

```diff
@@ -0,0 +1,23 @@
+"""Make building optional
+
+Revision ID: f046e456629f
+Revises: 3dafee32ba7d
+Create Date: 2022-02-03 15:24:28.785693
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'f046e456629f'
+down_revision = '3dafee32ba7d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('rooms', 'building', server_default='', schema='roombooking')
```

Comment:
I don't think this is needed; we aren't using server_default in the models either...